packages/http-client-java/generator/http-client-generator-mgmt/src/main/java/com/microsoft/typespec/http/client/generator/mgmt/mapper/FluentChoiceMapper.java

```diff
@@ -18,6 +19,6 @@ public static FluentChoiceMapper getInstance() {
 
     @Override
     public IType map(ChoiceSchema enumType) {
-        return MapperUtils.createEnumType(enumType, true, false);
+        return MapperUtils.createEnumType(enumType, true, false, "getValue", null, ClassType.STRING);
```

Comment:
I assume this is for ARM + Swagger? Which class has "getValue"?

Reply:
Yes.
This is related to [this comment](https://github.com/microsoft/typespec/pull/4492/files#r1772564418), for stream-style serialization.